modules/sandboxed-containers-uninstalling-kata-runtime.adoc

```diff
@@ -0,0 +1,18 @@
+//Module included in the following assemblies:
+//
+// *disabling-sandboxed-container-workloads.adoc
+
+[id="sandboxed-containers-uninstalling-kata-runtime_{context}"]
+
+= Uninstalling Kata Runtime
+
+This section describes how to remove and uninstall the `kata` runtime and all
+its related resources (for example, CRI-O config, RuntimeClass, and so on) from
+your cluster.
+
+.Procedure
+
+1. Delete the `KataConfig` custom resource using `oc delete kataconfig <KataConfig_CR_Name>`.
```

Comment:
Is this procedure a WIP? If it will be a one-step task, use a bullet. If there are more steps coming later, use `.` rather than numbering explicitly.